MarkerMetro.Unity.WinLegacyUnity/System/Security/Cryptography/EncryptionProvider.cs

```diff
@@ -94,10 +94,10 @@ public static string Encrypt(string toEncrypt, string key)
 
                 return strEncrypted;
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                // MetroEventSource.Log.Error(ex.Message);
-                return "";
+                System.Diagnostics.Debug.WriteLine(ex.Message);
+                throw new Exception("[EncryptionProvider] Error Encrypting a string");
```

Comment:
Is this better than throwing the original exception? Isn't better to use `new Exception(string, Exception)`?